src/field_info.rs

```diff
@@ -291,7 +371,7 @@ impl SetterSettings {
                     Err(Error::new_spanned(expr, "Expected simple identifier".to_owned()))
                 }
             }
-            _ => Err(Error::new_spanned(expr, "Expected (<...>=<...>)")),
+            _ => Err(Error::new_spanned(expr, "Expected (<...>=<...>) or (<...>(<...>))")),
```

Comment:
Maybe using [Lookahead1](https://docs.rs/syn/1.0.59/syn/parse/struct.Lookahead1.html) would make sense here, since that can generate a similar error message automatically. The syntax isn't as clean as the match statement here, though.